hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/AbstractScheduleHandler.java

```diff
@@ -211,9 +212,11 @@ protected ResponseCodeEnum validate(
             if (scheduleToValidate.hasScheduledTransaction()) {
                 if (!scheduleToValidate.executed()) {
                     if (!scheduleToValidate.deleted()) {
-                        Timestamp expiration = scheduleToValidate.calculatedExpirationTime();
+                        long expiration = scheduleToValidate.calculatedExpirationSeconds();
                         final Instant calculatedExpiration =
-                                ScheduleUtility.instantFromTimestamp(expiration, Instant.MAX);
+                                (expiration == Schedule.DEFAULT.calculatedExpirationSeconds()
+                                        ? Instant.ofEpochSecond(expiration)
+                                        : Instant.MAX);
```

Comment:
Is this right? I would think that if the expiration == default expiration seconds, it would be `Instant.MAX` (so flip the 'if' condition). Could be wrong, this just struck me as odd

Reply:
Great catch! I fixed it.

Reply:
@jsync-swirlds could you please look into why flipping this == to != did not cause a test to fail?

Reply:
The Handler methods are not tested yet, and PreHandle tests are incomplete.
Checking the expirations will be in those tests once written.